js/index.js

```diff
@@ -40,3 +40,134 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let nav = document.querySelectorAll('a');
+nav.forEach(function(value, i){
+  value.textContent = siteContent.nav[`nav-item-${i+1}`]
+})
+
+nav.forEach(navEle => {
+  navEle.style.color = 'green';
+})
+// __________New Navs_______________
+
+const navEl = document.querySelector('nav');
+
+const newNavItem1 = document.createElement('a')
+newNavItem1.textContent = "firstNewNav"
+
+navEl.appendChild(newNavItem1)
+
+const newNavItem2 = document.createElement('a')
+newNavItem2.textContent = "secondNewNav"
+
+navEl.appendChild(newNavItem2)
```

Comment:
You were supposed to use `appendChild()` and `prepend()` to add two new items to the nav.  Look into what `prepend() does compared to `appendChild()` and make sure you are familiar with both.